src/main/java/com/starkbank/CorporateCard.java

```diff
@@ -0,0 +1,828 @@
+     * <p>
+     * Return:
+     * @return CorporateCard object with updated attributes
+     * @throws Exception error in the request
+     */
+    public static CorporateCard get(String id, Map<String, Object> params) throws Exception{
+        return CorporateCard.get(id, params, null);
+    }
+
+    /**
+     * Retrieve a specific CorporateCard
+     * <p>
+     * Receive a single CorporateCard object previously created in the Stark Bank API by its id
+     * <p>
+     * Parameters:
+     * @param id [string]: object unique id. ex: "5656565656565656"
+     * @param user [Organization/Project object, default null]: Organization or Project object. Not necessary if starkbank.Settings.user was set before function call
+     * <p>
+     * Return:
+     * @return CorporateCard object with updated attributes
+     * @throws Exception error in the request
+     */
+    public static CorporateCard get(String id, User user) throws Exception{
+        return CorporateCard.get(id, null, user);
+    }
+
+    /**
+     * Retrieve a specific CorporateCard
+     * <p>
+     * Receive a single CorporateCard object previously created in the Stark Bank API by its id
+     * <p>
+     * Parameters:
+     * @param id [string]: object unique id. ex: "5656565656565656"
+     * <p>
+     * Return:
+     * @return CorporateCard object with updated attributes
+     * @throws Exception error in the request
+     */
+    public static CorporateCard get(String id) throws Exception{
+        return CorporateCard.get(id, null, null);
+    }
+
+    /**
+     * Update CorporateCard entity
+     * <p>
+     * Update a CorporateCard by passing id.
+     * <p>
+     * Parameters:
+     * @param id [string]: CorporateCard id. ex: "5656565656565656"
+     * @param patchData map of parameters
+     * status [string]: You may block the CorporateCard by passing 'blocked' in the status
+     * pin [string, default ""]: You may unlock your physical card by passing its PIN. This is also the PIN you use to authorize a purchase.
+     * displayName [string, default ""]: card displayed name
+     * rules [list of CorporateRule or HashMap, default null]: list of new CorporateRules. If the rule id isn't set, a new rule will be created.
+     * tags [list of strings, default null]: list of strings for tagging
+     * <p>
+     * Return:
+     * @return CorporateCard object with updated attributes
+     * @throws Exception error in the request
+     */
+    public static CorporateCard update(String id, Map<String, Object> patchData) throws Exception {
+        return CorporateCard.update(id, patchData, null);
+    }
+
+    /**
+     * Update CorporateCard entity
+     * <p>
+     * Update a CorporateCard by passing id.
+     * <p>
+     * Parameters:
+     * @param id [string]: CorporateCard id. ex: "5656565656565656"
+     * @param patchData map of parameters
+     * status [string]: You may block the CorporateCard by passing 'blocked' in the status
```

Comment:
You may block the CorporateCard by passing 'blocked' or activate by passing 'active' in the status